src/vm/readytoruninfo.cpp

```diff
@@ -671,7 +671,32 @@ PCODE ReadyToRunInfo::GetEntryPoint(MethodDesc * pMD, BOOL fFixups /*=TRUE*/)
             m_entryPointToMethodDescMap.InsertValue(PCODEToPINSTR(pEntryPoint), pMD);
     }
```

Comment:
Thanks. I'm now trying to figure out the build break in crossgen will update with your feedback.